.github/workflows/docs-netlify.yaml

```diff
@@ -0,0 +1,37 @@
+name: Publish docs to Netlify 
+
+on:
+  release:
+    types: [published] # remove auto deploying docs everytime pull request is merged
+  workflow_dispatch:
+
+permissions:
+  contents: write 
+
+jobs:
+  publish-docs:
+    name: Publish docs to Netlify
+    if: ${{ !github.event.release.prerelease }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: website
+    steps:
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+      - uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3.8.1
+        with:
+          node-version: "16.x"
+      - name: Trigger Netlify deployment
+        env:
+          USE_SSH: true
+          GIT_USER: git
+          GA_KEY: ${{ secrets.GA_KEY }}
+          ALGOLIA_API_KEY: ${{ secrets.ALGOLIA_API_KEY }}
+          NODE_OPTIONS: "--max-old-space-size=4096"
+        run: | 
+          git config --global user.email "corporate-it+weaveworks-docs-bot@weave.works"
+          git config --global user.name "weaveworks-docs-bot"
+          git switch docs-production
```

Comment:
I suppose the way you imagine hotfixes to the docs to land on the website is to issue PRs against `docs-production` which is ok in my opinion. That way we can use `main` as dev branch for the next release. But we need to cleary document this new process.